dev-client/src/screens/SiteNotesScreen/components/SiteNoteCard.tsx

```diff
@@ -36,6 +38,13 @@ export const SiteNoteCard = ({note}: Props) => {
   const {t} = useTranslation();
   const navigation = useNavigation();
 
+  const currentUser = useSelector(state => state.account.currentUser.data);
+  const currentUserIsAuthor = note.authorId === currentUser?.id;
+  const siteRole = useSiteRoleContext();
+  const currentUserIsManager =
```

Comment:
This isn't a good variable name, but absent a full overhaul of how the FE models permissions to match the BE and product team documents it seemed to mostly capture the spirit.